api/include/opentelemetry/context/context_value.h

```diff
@@ -20,4 +20,4 @@ using ContextValue = nostd::variant<bool,
                                     nostd::shared_ptr<trace::Span>,
                                     nostd::shared_ptr<trace::SpanContext>>;
 }  // namespace context
-OPENTELEMETRY_END_NAMESPACE
+OPENTELEMETRY_END_NAMESPACE
```

Comment:
curious - why is this changed?

Reply:
Please revert this change.

Reply:
I don't believe this file should be changed. Can you add the whitespace back to the end of the file?